walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -121,25 +121,27 @@ public int compare(Event a, Event b) {
    * of event) or (end of event - end of slot) are at least as long as {@code reqDuration}
    * they are kept in {@code curSlots} as possible meeting times.
```

Comment:
Update this part of the description since switching to newSlots/removeSlots version.